charts/edge/templates/statefulset.yaml

```diff
@@ -100,6 +100,10 @@ spec:
           - name: MEZMO_MEM_BUFFER_MAX_EVENTS
             value: {{ .Values.maxMemBufferEvents }}
           {{- end }}
+          - name: REDUCE_BYTE_THRESHOLD_PER_STATE
+            value: {{ .Values.reduceByteThresholdPerState }}
```

Comment:
Found out we need to quote these values. Also the two preexisting ones above them.